opm/parser/eclipse/Units/ConversionFactors.hpp

```diff
@@ -133,6 +133,7 @@ namespace Opm {
             /// \name Force
             /// @{
             constexpr const double Newton = kilogram*meter / square(second); // == 1
+            constexpr const double dyn    = 1e-5*Newton; // cf. wikipedia
```

Comment:
I just noticed this:

https://github.com/OPM/opm-parser/blob/master/opm/parser/eclipse/Units/ConversionFactors.hpp#L116

do you still want to change the comment? both?